cupy/testing/helper.py

```diff
@@ -101,8 +101,13 @@ def test_func(self, *args, **kw):
             # values are negative.
             if _contains_signed_and_unsigned(kw):
                 inds = _make_positive_indices(self, impl, args, kw)
-                cupy_result = cupy.asnumpy(cupy_result)[inds]
-                numpy_result = cupy.asnumpy(numpy_result)[inds]
+                try:
```

Comment:
Sorry my code has bug.
This is fixed version.
https://github.com/pfnet/chainer/pull/1784/commits/8f85f6249b4750ba34b8f168020fa9e67b8f9289